staging/src/k8s.io/client-go/discovery/OWNERS

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - apelisse
+  - seans3
```

Comment:
Same question as I asked Antoine.  Do you have changes in mind or just maintenance? If you're asking because you want to make changes, I'd like to make the changes first and then hand over the keys. If you're thinking about maintenance plus aggregated discovery, how about we get aggregated discovery to beta and then add this?

Reply:
Waiting sound fine. Closing this PR.

More Context: There is a plan to significantly simplify the discovery client code at a time when only the new discovery format is served. This is pretty far in the future, but it's probably a good idea to keep our group in the loop on any changes in this discovery client code in the mean time. Thanks.